make/project.mk

```diff
@@ -440,6 +440,7 @@ CFLAGS ?=
 EXTRA_CFLAGS ?=
 CFLAGS := $(strip \
 	-std=gnu99 \
+  -fplan9-extensions \
```

Comment:
FWIW, you can achieve this change without patching IDF by setting `EXTRA_CFLAGS := -fplan9-extensions` in your project Makefile, before the line that includes `project.mk`